at_secondary/at_secondary_server/lib/src/verb/handler/scan_verb_handler.dart

```diff
@@ -122,7 +116,10 @@ class ScanVerbHandler extends AbstractVerbHandler {
           key.toString().startsWith('privatekey:') ||
           key.toString().startsWith('public:_') ||
           key.toString().startsWith('private:'));
-      keyString = keys.toString();
+      for (var key in keys) {
+        var modifiedKey = key!.replaceAll('${atConnectionMetadata.fromAtSign}:', '');
```

Comment:
Explain purpose of this replaceAll, it's not clear unless you happen to know lots of details from elsewhere about keys in the @ protocol.